dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/interceptor/ServiceInvokerInterceptor.java

```diff
@@ -71,6 +71,11 @@ public void run() {
                         exchange.setOutMessage(outMessage);
                     }
                     copyJaxwsProperties(message, outMessage);
+                }
+
```

Comment:
can you add a comment here indicating that this is a liberty-specific change?  the comment should also state why the invoke is done later.  it sounds like this might be a liberty-specific problem, in which case, we shouldn't contribute this fix back to the CXF community.